External/Plugins/ASCompletion/Model/MemberModel.cs

```diff
@@ -538,4 +538,40 @@ public int Compare(MemberModel a, MemberModel b)
         }
 
     }
+
+    /// <summary>
+    /// Compare members based on import name
+    /// </summary>
+    public class ByImportTypeMemberComparer : IComparer<MemberModel>
+    {
+        public static Int32 CompareImports(String import1, String import2)
+        {
+            // Use case-insensitive comparison.
+            //IComparer cmp = StringComparer.OrdinalIgnoreCase;
+            // Use case-sensitive comparison.
+            IComparer cmp = StringComparer.Ordinal;
```

Comment:
You should make the code more efficient by finding index of `.` separating package and type and doing (up to) 2 comparisons for package and type.

There are a few edge cases to consider for Haxe, here are valid imports:

    import foo.bar.ModuleName;
    import foo.bar.ModuleName.TypeName;

As algorithm I'd do: 

- starting from 0, get next index of `.`
- if index < 0 previous index is separation
- if char after `.` is uppercase (I think there's `Char.IsUpperCase(c)`) then stop (this takes care of Haxe modules)
- else try getting next `.` and repeat